asyncwrapper/pom.xml

```diff
@@ -17,7 +17,7 @@
 		<java.version>1.8</java.version>
 		<kotlin.compiler.jvmTarget>1.8</kotlin.compiler.jvmTarget>
 		<pulsar.version>2.10.1</pulsar.version>
-		<wpsclient.version>1.1.1</wpsclient.version>
+		<wpsclient.version>1.1.2</wpsclient.version>
```

Comment:
Works!
This has `get_tsunamap` execute successfully and stores data correctly in filestorage.